src/hangman/games.service.ts

```diff
@@ -33,9 +32,7 @@ export class GamesService {
 
   async getAllGames(): Promise<{ count: number; games: GameProjection[] }> {
     this.logger.log('getAllGames');
-    const games = await this.gamesProjectionRepository.find({
-      order: { dateModified: 'DESC' },
-    });
+    const games = await this.queryBus.execute(new GetGameesQuery());
```

Comment:
nice, i hadn't looked into the queryBus yet. What's the advantage ?

Reply:
I don't know to be honest. Just following the other CQRS example.